srcpkgs/vala-panel-appmenu/template

```diff
@@ -0,0 +1,91 @@
+# Template file for 'vala-panel-appmenu'
+pkgname=vala-panel-appmenu
+version=0.7.1
+revision=1
+build_style=cmake
+configure_args="-DENABLE_XFCE=ON -DENABLE_VALAPANEL=ON
+ -DENABLE_MATE=ON -DENABLE_JAYATANA=ON -DENABLE_APPMENU_GTK_MODULE=ON
+ -DJAVA_INCLUDE_PATH=/usr/lib/jvm/openjdk/include
+ -DJAVA_JVM_LIBRARY=/usr/lib/jvm/openjdk/lib
+ -DJAVA_AWT_LIBRARY=/usr/lib/jvm/openjdk/lib/amd64"
+hostmakedepends="pkg-config"
+makedepends="vala-devel bamf-devel gtk+-devel gtk+3-devel glib-devel
+ libxkbcommon-devel openjdk libpeas-devel libdbusmenu-glib-devel cmake-vala
+ xfce4-panel-devel xfconf-devel libmate-panel-devel libXt-devel"
+depends="glib bamf "
```

Comment:
extra whitespace

Reply:
also will problably be found automatically

Reply:
should i remove them then?